src/components/dashboard/Dashboard.tsx

```diff
@@ -300,8 +300,8 @@ export const Dashboard: React.FunctionComponent<DashboardProps> = ({
       </div>
     )
   } else {
-    if (error !== undefined) {
-      return <Alert severity="error">{error!['message'] || error}</Alert>
+    if (error && error !== undefined) {
+      return <Alert severity="error">{error['message'] || error}</Alert>
     }
```

Comment:
If error is `null` this will be true--not sure if that is possible

```suggestion
    if (error && error != null) {
      return <Alert severity="error">{error['message'] || error}</Alert>
    }
```